app/kumactl/cmd/apply/apply.go

```diff
@@ -40,7 +41,7 @@ func NewApplyCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 		Use:   "apply",
 		Short: "Create or modify Kuma resources",
 		Long:  `Create or modify Kuma resources.`,
-		RunE: func(cmd *cobra.Command, args []string) error {
+		RunE: kumactl_errors.FormatErrorWrapper(func(cmd *cobra.Command, args []string) error {
```

Comment:
Let's try not to use `kumactl_errors.FormatErrorWrapper` explicitly.

Instead, let's traverse the command tree and wrap all non-nil `*RunE` handlers, such as
* `PersistentPreRunE`
* `PreRunE`
* `RunE`
* `PostRunE`
* `PersistentPostRunE`

We could update `func NewRootCmd()` to do such a wrapping in the end.